app/views/application/_footer.html.erb

```diff
@@ -42,42 +42,44 @@
-            <a class="footer-secondary__link footer-secondary__sfgb-link" href="https://singlefinancialguidancebody.org.uk/" target="_blank">
-              <span class="footer-secondary__sfgb-text">
-                <span><%= t('footer_secondary.sfgb.part_of') %></span>
-                <span><strong><%= t('footer_secondary.sfgb.single_financial') %></strong></span>
-                <span><strong><%= t('footer_secondary.sfgb.guidance_body') %></strong></span>
-              </span>
-              <%= image_tag('sfgb-logo.png', class: 'footer-secondary__sfgb-logo', alt: "#{t('footer_secondary.sfgb.single_financial')} #{t('footer_secondary.sfgb.guidance_body')} logo") %>
-              <span class="visually-hidden"><%= t('footer_secondary.sfgb.hidden_text') %></span>
-            </a>
-          </li>
-        </ul>
+    <div class="l-constrained">
+      <div class="l-footer-secondary--inner">
+        <div class="l-footer-secondary__list--left">
+          <ul class="l-footer-secondary__list">
+            <li class="l-footer-secondary__list-item  l-language-link">
+              <a class="t-cy-link" href="https://www.moneyadviceservice.org.uk/cy/empty" lang="cy">
+                <span class="icon icon--globe"></span>
+                Cymraeg
+              </a>
+            </li>
+            <li class="l-footer-secondary__list-item accessibility-feedback-link">
+              <a class="footer-secondary__link" href="http://accessin.org/usersubmit_mas.php?param=nojscript"
+                 onclick="window.open('http://accessin.org/usersubmit_mas.php?param=' + window.location,'',
+                     'resizable=yes,location=no,scrollbars=yes,toolbar=no,left=20,top=0,height=900,width=800');
+                     return false;" title="Please use this button to tell us if you have an accessibility issue on our website"
+                 target="_blank">
+                <span class="icon icon--accessibility"></span><%= t('footer_secondary.accessin.button') %>
+                <span class="visually-hidden"><%= t('footer_secondary.accessin.hidden_text') %></span>
+              </a>
+            </li>
+            <li class="l-footer-secondary__list-item footer-secondary__mps-link">
+              <a href="https://moneyandpensionsservice.org.uk/" target="_blank">
+                <span class="footer-secondary__mps-text">
+                  <span><%= t('footer_secondary.mps_provided_1') %></span>
+                  <span><%= t('footer_secondary.mps_provided_2') %></span>
+                  <span><%= t('footer_secondary.mps_provided_3') %></span>
+                </span>
+                <%= image_tag("mps-logo-#{locale}.png", class: 'mps-logo__img', alt: "#{t('footer_secondary.mps')} logo") %>
+                <span class="visually-hidden">opens in a new window</span>
+              </a>
+            </li>
+          </ul>
+        </div>
       </div>
       <div class="l-footer-secondary__list--right">
```

Comment:
This container has inconsistent horizontal margins with its sibling. Either this needs to have the same margins or (maybe better) the parent of both to have the margins applied to it.

Reply:
Additionally this needs come further work for the widest viewports so that it does not display below its sibling.